src/acceptance/java/io/codurance/pouch/GetResourceFeature.java

```diff
@@ -0,0 +1,4 @@
+package io.codurance.pouch;
+
+public class GetResourceFeature {
+}
```

Comment:
Oops, no! I didn't commit this file!! That's what I was surprised about at the beginning of our call. I checked out a new feature branch, created a new file, then switched back to master and was surprised the file was still there. Then checked out a new branch for this PR here and committed the files we touched, not that new acceptance test.

Reply:
Right, so unless it was added+committed to the branch, that file will stay with you. Because you never committed it, it was still in your workspace. Worth always doing a quick diff in your IDE of all your changes before doing a commit. I invariably find little things lurking when I do this.

Reply:
How can I remove this file from this PR? Wait...I'm gonna delete it. It was just something I started between lunch and our call.

Reply:
Provided you are on a feature branch:
```
git rm path/to/file
git commit --amend
```

Reply:
Ah, I see. I have deleted the file already. The build fails because of:
```
* What went wrong:
Execution failed for task ':deployHeroku'.
> org.apache.http.client.HttpResponseException: Too Many Requests
```
So I have to wait for a bit to restart the build. I think Travis CI and/or Heroku will blacklist me soon for spamming their services in a free tier :-/

Reply:
We're in the green now! =D